src/main/scala/org/apache/spark/sql/delta/storage/BaseExternalLogStore.scala

```diff
@@ -144,10 +165,8 @@ abstract class BaseExternalLogStore(sparkConf: SparkConf, hadoopConf: Configurat
       return listedFromFs.iterator
     }
 
-    val listedFromDB = listFromCache(fs, resolvedPath)
-      .toList
-      .map(entry => if (!entry.isComplete) tryFixTransactionLog(fs, entry) else entry)
```

Comment:
I don't understand this change. Are we going to stop fixing transactions on read?

Reply:
No, of course. Is problem after rebase, my mistake